### PR TITLE
Tal linkdin

### PR DESCRIPTION
**Pull request by tal556**
linkdin finish - jobs page before css

### CODE REVIEW

File: client/src/components/LinkdIn.tsx [hunk 1]
```diff
@@ -1,103 +1,119 @@
...
+import { Input, Row, Col,Spin } from "antd";
+import LinkdInRouter from "./LinkdInRouter";
+import { useParams } from "react-router-dom";
 
+export default function JobsList() {
 const { Search } = Input;
+const [fetchedDataDb, setfetchedDataDb] = useState<any>([])
+const [UserInfo, setUserInfo] = useState([]);
 
-export default function JobsList() {
```
Comment (orgolestani) [inline client/src/components/LinkdIn.tsx:11]: don't forget to remove unnecessary comments and logs

File: client/src/components/LinkdIn.tsx [hunk 2]
```diff
@@ -1,103 +1,119 @@
...
+    }
+   // getManyJobOffers({isHidden:false}).then((data)=>{console.log(data)})
+   fetchData()
+ }, []);
+
+//  console.log(fetchedDataDb);
+
+// production level MockUp
+
+const CardsData: any = [
```
Comment (orgolestani) [inline client/src/components/LinkdIn.tsx]: if fetch data works, Then you can delete this const, right?

**Review by orgolestani — APPROVED** [review]
a bit messy, but great job! would appriciate if we can try and make it cleaner

**Comment by jamberusimantov** [comment]
send an updated one

**Pull request closed without merging**